fplus-lib/src/core/mod.rs

```diff
@@ -1970,24 +1977,32 @@ impl LDNApplication {
         issue_number: String,
         info_owner: String,
         info_repo: String,
-        db_model: ApplicationModel,
+        db_model: Option<ApplicationModel>,
     ) -> Result<bool, LDNError> {
-        let gh = github_async_new(info_owner.clone(), info_repo.clone()).await;
-    
-        let comment = if db_model.owner == info_owner && db_model.repo == info_repo {
-            // Application already exists in the same repository
-            format!(
-                "This wallet address already exists in another application: #{}",
-                db_model.pr_number
-            )
-        } else {
-            // Application exists in a different repository
-            format!(
-                "This wallet address already exists in another application: http://github.com/{}/{}/issues/{}",
-                db_model.owner, db_model.repo, db_model.pr_number
-            )
-        };
+        let mut comment = "This wallet address has already received datacap in other application".to_string();
```

Comment:
```suggestion
        let mut comment = "The wallet address in this application has previously received datacap from another source. Please update the application to use a new client wallet address, so that it is clear that datacap usage is associated with this application.".to_string();
```